src/rpc/api.ts

```diff
@@ -448,4 +448,17 @@ export namespace Api {
     transactionCount: string; // uint32
     ledgerCount: number; // uint32
   }
+
+  export interface ContractBalanceResponse {
+    latestLedger: number;
+    // present only on success, otherwise request malformed or no balance
+    trustline?: {
```

Comment:
Can this be called `contractBalance` or something like that instead? Calling it a trustline is misleading. The comment applies to line 992 as well.

Reply:
Valid, can do! Changed to `balanceEntry` and `amount` to match the truth more in bb7771ba